JournalCoreData/View Controllers/EntriesTableViewController.swift

```diff
@@ -93,6 +93,8 @@ class EntriesTableViewController: UITableViewController, NSFetchedResultsControl
         case .delete:
             guard let indexPath = indexPath else { return }
             tableView.deleteRows(at: [indexPath], with: .automatic)
+        @unknown default:
+            fatalError()
```

Comment:
Perhaps just `break` in this scenario so the app won't crash for the user if Apple ever adds any cases.